app/build.gradle

```diff
@@ -60,6 +60,8 @@ dependencies {
     implementation 'com.squareup.retrofit2:converter-moshi:2.9.0'
     // Coil dependency
     implementation "io.coil-kt:coil:1.1.1"
+    // Picasso dependency
```

Comment:
usually it's useless to add such comments. We know that the dependency is picasso because its name is picasoo :)